source/activity/view-logs.txt

```diff
@@ -101,6 +112,7 @@ The following types are valid:
 - ``graphql``
 - ``sync``
 - ``schema``
+- ``error_handler``
```

Comment:
need to confirm this once it's updated in CLI

Reply:
Just to close the loop a bit -- this ticket for this work will be in the next release: https://jira.mongodb.org/browse/BAAS-28612

Reply:
updated to `trigger_error_handler` throughout